src/app/app.component.ts

```diff
@@ -31,13 +31,17 @@ export class AppComponent {
         let graphData = [];
         for (let location of definition.locations) {
           // Sometimes people make typos -- don't bail, return what you can.
-          if (!data[location]) continue;
+          if (!data[location]) {
```

Comment:
check out my PR! I kind of handle this, but silently. I wonder if the correct behavior here is to just trim it out of the url automatically.

Reply:
Could be nice to trim it automatically yeah!

Whether we trim or not, I think it's a nice usability thing to give a warning or something (whether in the UI or as simple as `console.warn` here) when trimming.